### PR TITLE
Revise footer and styling fixes

### PR DESCRIPTION
**Pull request by coreycaitlin**
Fixes issue(s) #1940 and #2064.

[CircleCI](https://circleci.com/gh/18F/doi-extractives-data/tree/footer-fix)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/footer-fix/)

Changes proposed in this pull request:

- remove sitemap from footer
- fix styling bug on social link (with an assist from @meiqimichelle)
- fix disbursements table styling

/cc @meiqimichelle @ericronne

### CODE REVIEW

**Comment by ericronne** [comment]
Footer 👍 

Table could use a smidge of a tweak, if poss ...

**A** Left align headers
**B** Increase padding above and below content, so the rules aren't so close to it (i bumped from 5 to 10 here)

(Megachart is a good reference here)

image

thx!

File: _sass/elements/_tables.scss [hunk 1]
```diff
@@ -54,6 +54,7 @@ td {
 
   thead {
     background-color: transparent;
+    line-height: 1.2;
```
Comment (gemfarmer) [inline _sass/elements/_tables.scss]: This could be `line-height: $light-height-caps;`

It will compute to `line-height: 1.2rem;`

**Comment by coreycaitlin** [comment]
- Fixed top/bottom padding in the table
- Left-aligned headers
- Moved the `$line-height-caps` variable to the variables file so it's easier to find next time! Also, this made it stop failing to build.

Hopefully good to go now!

**Comment by gemfarmer** [comment]
LGTM!

**Comment by ericronne** [comment]
👍

**Pull request merged**